File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1182.lua
soma=0
l = io.read("*number")
carro = io.read()
escolha = io.read()
for i=0,11 do
    for j=0,11 do
        valor = io.read("*number")
        if(j==l) then
            soma=soma+valor
        end
    end
end
    
if(escolha=="S") then
    print(string.format("%.1f", (soma)));
else
    print(string.format("%.1f", (soma/12)));
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1182.py
soma=0
l = int(input())
escolha = input()
for i in range(12):
    for j in range(12):
        valor = float(input())
        if(j==l):
            soma=soma+valor
    
if(escolha=="S"):
    print("{:.1f}".format(soma))
else:
    print("{:.1f}".format(soma/12))